logger/mask.go

```diff
@@ -159,6 +204,31 @@ func MaskJSON(json, maskChar string, toMasks []MaskData) string {
 	return string(out)
 }
 
+func maskRecursive(obj *fastjson.Value, keys []string, maskChar string, toMask MaskData) {
```

Comment:
do you think it is better to rename this to `maskArrayRecursive`? (since it is used only for masking array)

Reply:
Done in 2b2bb0c.